packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -3337,6 +3372,21 @@ class SemanticsConfiguration {
     _isSemanticBoundary = value;
   }
 
+  /// Whether to block pointer related user actions for the rendering subtree.
+  ///
+  /// Setting this to true will prevent users from interacting with this
+  /// rendering object and its subtree through pointer-related
```

Comment:
nit: this is not a rendering object. Maybe say "with the rendering object that produces this semantics configuration"